### DISCUSSION TITLE
Add and Remove Inputs

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#10219] [CLI - PWA mode] [answered]
Hello Friends

Am trying to implement a component that allows me to add input fields when i click on the add button and remove input fields when I click on the remove button. See example below 

![image](https://user-images.githubusercontent.com/84211129/127273912-7608a983-061a-4023-ab68-8208887e52b0.png)

I have try to implement it but its now working correctly , can someone please help 
https://codepen.io/akashramtohul4/pen/ZEKrZQL

Thanks

**Comment by metalsadman** [comment, marked as answer]
@akashramtohul4 just bugs in your code, I would suggest not mixing options api and composition api. https://codepen.io/metalsadman/pen/wvdybge?editors=1010
